src/main/java/org/ossgang/commons/properties/WrapperProperty.java

```diff
@@ -58,4 +60,10 @@ public T get() {
     public Subscription subscribe(Observer<? super T> listener, SubscriptionOption... options) {
         return observableValue.subscribe(listener, options);
     }
+
+    @Override
+    public Transition<T> accumulate(T x, BinaryOperator<T> accumulatorFunction) {
```

Comment:
I don't see it possible, at least not with reasonable guarantees.

What I could imagine though: We could leave the "Property" interface as it was, and introduce a new interface "AtomicProperty" that extends it and has all `accumulate` and all the atomic sugar methods.

I think this would be nice for a few reasons:
- basic "Property" stays a clean definition of "something you can set/get/subscribe" with no implications of whether these operations are atomic or not
- the default methods are nicely separated from the main Property interface
- WrapperProperty can implement `Property` (thus no accumulate) while SimpleProperty can implement `AtomicProperty`
- AtomicProperty is very explicit and close to AtomicReference :-)

What do you think?